### PR TITLE
Added pipeline creation functions

### PR DESCRIPTION
**Pull request by kylesayrs**
This PR adds creation functions that allow easier pipeline creation. The new functions positional arguments and keyword arguments to be passed.

### CODE REVIEW

File: src/deepsparse/pipeline.py [hunk 1]
```diff
@@ -49,6 +54,51 @@
 _REGISTERED_PIPELINES = {}
 
 
+def question_answering_pipeline(*args, **kwargs) -> "Pipeline":
+    """
+    :param args: the positional arguments to pass into Pipeline.create
+    :param kwargs: the keyword arguments to pass into Pipeline.create
+    :return: question_answering pipeline created with args and kwargs
+    """
+    return Pipeline.create(*(("question_answering",) + args), **kwargs)
```
Comment (rahul-tuli) [inline src/deepsparse/pipeline.py:63]: This is so cool

**Review by rahul-tuli — DISMISSED** [review]

File: src/deepsparse/pipeline.py [hunk 2]
```diff
@@ -49,6 +54,51 @@
...
+
+
+def text_classification_pipeline(*args, **kwargs) -> "Pipeline":
```
Comment (bfineran) [inline src/deepsparse/pipeline.py:66]: let's make another one called `sentiment_analysis_pipeline` that does the same thing

File: src/deepsparse/pipeline.py (hunk 1 above)
Comment (bfineran) [inline src/deepsparse/pipeline.py:63]: any reason we can't do `Pipeline.create("question-answering", *args, **kwargs)` a little easier to read

**Review by bfineran — CHANGES_REQUESTED** [review]
looks good overall @kylesayrs.

it's nice that using generic args, keyword args for these will limit the impact downstream updates causing future issues if these functions aren't updated, let's at least get the existing docstrings for each of these pipelines copied in though, so when docs are generated, each of these functions will have some info on what should be included

**Review by bfineran — APPROVED** [review]

**Review by rahul-tuli — APPROVED** [review]

**Pull request merged**